src/contrib/coding-style.rst

```diff
@@ -1,12 +1,494 @@
 Coding Style
+############
+
+Introduction
 ============
+This manual contains coding standards and guidelines for SSSD contributors.
+All new code should adhere to these standards. Those rules does not apply to
```

Comment:
s/Those rules does not/These rules do not/

Reply:
Fixed